jvcproj/plugin.yaml

```diff
@@ -0,0 +1,25 @@
+# Metadata for the Smart-Plugin
+plugin:
+    # Global plugin attributes
+    type: interface                 # plugin type (gateway, interface, protocol, system, web)
+    description:
+        de: 'Plugin um JVC D-ILA Projektoren zu kontrollieren und durch jvcprojectortools erstellte Gammakurven zu uebertragen'
+        en: 'plugin to control JVC D-ILA projectors and transfer gammatables generated with jvcprojectortools'
+    maintainer: NPortz
+    tester: 
+#    keywords: iot xyz
+#    documentation: https://github.com/smarthomeNG/smarthome/wiki/CLI-Plugin        # url of documentation (wiki) page
+#    support: https://knx-user-forum.de/forum/supportforen/smarthome-py
+
+    version: 1.0.0                 # Plugin version
+    sh_minversion: 1.1             # minimum shNG version to use this plugin
```

Comment:
I guess SH 1.1 does not have SmartPlugin support. I would increase at least to 1.3.

Reply:
Yes, thats right! The complete plugin.yaml has to be revised anyway.